tests/integration-tests/tests/update/test_update.py

```diff
@@ -444,76 +439,3 @@ def get_batch_spot_bid_percentage(stack_name, region):
         .get("computeResources")
         .get("bidPercentage")
     )
-
-
-def _check_update_compute(
-    pcluster_config_reader,
-    clusters_factory,
-    init_config_name,
-    update_config_name,
-    scheduler,
-    region,
-    disable_hyperthreading=False,
-    cfn_scheduler_slots=None,
-):
-    # Create cluster with initial configuration
-    init_config_file = pcluster_config_reader(config_file=init_config_name)
-    cluster = clusters_factory(init_config_file)
-
-    # Check cfn_scheduler_slots before changing compute instance type
-    _check_compute_node_slots(
-        cluster,
-        scheduler,
-        region,
-        "c5.xlarge",
-        disable_hyperthreading=disable_hyperthreading,
-        cfn_scheduler_slots=cfn_scheduler_slots,
-        nodes_number=2,
-    )
-    cluster.stop()
-
-    # Update cluster with new configuration and change the compute instance type to c5.2xlarge
-    updated_config_file = pcluster_config_reader(update_config_name)
-    cluster.config_file = str(updated_config_file)
-    cluster.update()
-
-    # Check cfn_scheduler_slots changed by changing compute instance type
-    _check_compute_node_slots(
-        cluster,
-        scheduler,
-        region,
-        "c5.2xlarge",
-        disable_hyperthreading=disable_hyperthreading,
-        cfn_scheduler_slots=cfn_scheduler_slots,
-    )
-
-
-def _check_compute_node_slots(
```

Comment:
Why are we removing these checks?

Reply:
This checks were used for sge and torque. Now this code are not used